### DISCUSSION TITLE
How to disable the user app ?

### DISCUSSION THREAD

**Discussion by thibautd** [piccolo-orm/piccolo#974] [General]
Hello,

I don't want to use the user application provider by Piccolo, I didn't add it to my `APP_REGISTRY` but it's still here. I have a boring warning when I use my own `User` table, and the user commands are displayed in `piccolo` CLI output.

Do you know how to remove it ?

**Comment by sinisaos** [comment]
@thibautd There is already a [discussion](https://github.com/piccolo-orm/piccolo/discussions/867) and a temporary solution for that. Hope that helps.

**Comment by thibautd** [comment]
Thanks. Indeed an option to disable this behavior would be nice.
